features/step_definitions/header.js

```diff
@@ -0,0 +1,15 @@
+const {defineSupportCode: cucumber} = require(`cucumber`);
+
+cucumber(({Given, Then}) => {
+
+  Given('I am viewing the dashboard', function (int, callback) {
```

Comment:
* take out the `int`
* I usually call `callback` **done** , but that's all good.
* You can also use a regular expression for this it would look like this
```javascript
Given(/I add the numbers (.*) and (.*)$/, function(firstNumber, secondNumber, done){
  // In this case I can save the result to the world global accessible throughout the other step definitions.
  this.result = Number(firstNumber) + Number(secondNumber); // This regex returns a string, so we turn em into numbers.
  done();
});
```

Reply:
Your gif usage is on another level, just btw.